tests/func/test_commit.py

```diff
@@ -83,6 +83,12 @@ def test_commit_no_exec(tmp_dir, dvc):
     assert dvc.status(stage.path) == {}
 
 
+def test_commit_granularity(tmp_dir, scm, dvc):
+    tmp_dir.dvc_gen("foo", "foo", commit="add files")
+    dvc.commit("foo")
+    assert dvc.status() == {}
```

Comment:
This test only tests the `collect_granular`, not the granularity of the `commit`.